components/button/overview.md

```diff
@@ -44,95 +39,54 @@ To add a Telerik Button to your Blazor app, use the `<TelerikButton>` tag:
 }
 ````
 
->caption The result from the code snippet above
-
-![](images/basic-button.png)
-
->caption Component namespace and reference
-
-````CSHTML
-@using Telerik.Blazor.Components
-
-<TelerikButton @ref="theButton">Hello!</TelerikButton>
-
-@code{
-	Telerik.Blazor.Components.TelerikButton theButton;
-}
-````
-
-## Primary Button
+To see the result from the code snippet above, select the **PREVIEW** tab in the code snippet toolbar.
 
-You can also make the button use a strong color to attact attention, called Primary button styling. To do that, set its `Primary` property to true.
+## Icons
 
->caption Button with the Primary color scheme from the current theme
+To visually communicate the purpose of a button, you can use add an image, a sprite, or font icon. You can choose between the wide range of built-in font icons or even use your custom font icons. [Read more about the Blazor Button icons...]({%slug button-icons%})
 
-````CSHTML
-<TelerikButton Primary="true">Primary</TelerikButton>
-````
+## Type
 
->caption The result from the code snippet above, with the Default theme
+To control the desired behavior of the Blazor Button, you can use the `type` attribute. This allows you to select on of the following button types: `Submit`, `Reset`, and `Button`. [Read more about the Blazor Button type...]({%slug button-type%})
 
-![](images/primary-button.png)
+## Events
 
+The Blazor Button generate events that you can handle and further customize the behavior of the Button. [Read more about the Blazor Button events...]({%slug button-events%}).
 
 ## Disabled State
 
-To disable a button, set its `Enabled` attribute to `false`.
-
->caption Disabled Telerik Button
-
-````CSHTML
-<TelerikButton Enabled="false">Disabled Button</TelerikButton>
-````
-
->caption Comparison between disabled and enabled button
-
-![](images/disabled-button.png)
+To stop the users from interacting with a button until certain requirements are met, you can render a disabled button. [Read more about the disabled Blazor Button...]({%slug button-disabled%}).
```

Comment:
Simplify

```suggestion
To prevent user interaction with a Button, disable it. [Read more about the disabled Blazor Button...]({%slug button-disabled%}).
```